lib/src/message.dart

```diff
@@ -26,13 +27,13 @@ abstract class Message {
   /// The value is immutable.
   final Map<String, String> headers;
 
-  /// Extra context that can be used by for middleware and handlers.
+  /// Extra context that can be used by middleware and handlers.
   ///
   /// For requests, this is used to pass data to inner middleware and handlers;
   /// for responses, it's used to pass data to outer middleware and handlers.
   ///
   /// Context properties that are used by a particular package should begin with
-  /// that package's name followed by a period. For example, if [logRequests]
+  /// that package's name followed by a period. For example, if `logRequests`
```

Comment:
Referring to `logRequests` here seems strange since that's not middleware that exists—and probably not something this package will ever provide. Maybe rephrase this as describing a hypothetical middleware?